src/lib/API/api.js

```diff
@@ -346,13 +346,6 @@ export class APIService {
     })
   }
 
-  /**
-   * `/verify/w3/bonuses` get api call
-   */
-  redeemBonuses() {
```

Comment:
as we're removing it totally, we need to also remove it from server (in a separate PR).
w3/bonuses endpoint and everything related (tests, utils, service methods)